infra/tools/ip/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "fendermint_infra_ip"
```

Comment:
I don’t think this needs a `fendermint_` prefix, it’s even outside the directories.